talk/owt/sdk/include/cpp/owt/base/globalconfiguration.h

```diff
@@ -81,6 +81,41 @@ class GlobalConfiguration {
           audio_frame_generator_.reset(nullptr);
       }
   }
+
+  /**
+   @brief This function sets the port ranges of different payload types.
+  */
+  static void SetPortRanges(int audio_min,
+      int audio_max,
+      int video_min,
+      int video_max,
+      int screen_min,
+      int screen_max,
+      int data_min,
+      int data_max) {
+    audio_min_ = audio_min;
+    audio_max_ = audio_max;
+    video_min_ = video_min;
+    video_max_ = video_max;
+    screen_min_ = screen_min;
+    screen_max_ = screen_max;
+    data_min_ = data_min;
+    data_max_ = data_max;
+  }
+
+  /**
+   @brief This function enables stream dump before decoder.
```

Comment:
Please describe where is the dumped file.

Reply:
done